src/network/http_server.rs

```diff
@@ -308,11 +308,11 @@ pub struct ReactionRequest {
 #[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct ReactionsByFidRequest {
     fid: u64,
-    #[serde(rename = "reactionType")]
+    #[serde(rename = "reactionType", alias = "reaction_type")]
     reaction_type: ReactionType,
-    #[serde(rename = "pageSize", skip_serializing_if = "Option::is_none")]
+    #[serde(rename = "pageSize", alias = "page_size", skip_serializing_if = "Option::is_none")]
     page_size: Option<u32>,
-    #[serde(rename = "pageToken", skip_serializing_if = "Option::is_none")]
+    #[serde(rename = "pageToken", alias = "page_token", skip_serializing_if = "Option::is_none")]
```

Comment:
I think we can just get rid of `rename` and `alias` and it should pick up the default snake case?

@CassOnMars Any reason we have to use camel case? Snake case is what hubs use and it's breaking backwards compatibility